include/aca_util.h

```diff
@@ -196,4 +197,44 @@ static inline bool aca_is_port_on_same_host(const std::string hosting_port_ip)
   return (rc == EXIT_SUCCESS);
 }
 
+static inline string aca_covert_cidr_to_netmask(const std::string cidr) {
+  if (cidr.empty()) {
```

Comment:
I think we should throw exception (catch by the caller) when cidr is empty or invalid.